x-pack/legacy/plugins/siem/public/components/ml_popover/jobs_table/jobs_table.tsx

```diff
@@ -38,7 +38,11 @@ const getJobsTableColumns = (
     name: i18n.COLUMN_JOB_NAME,
     render: ({ id, description }: Job) => (
       <JobNameWrapper>
-        <EuiLink href={`${chrome.getBasePath()}/app/ml`} target="_blank">
+        <EuiLink
+          data-test-subj="jobs-table-link"
+          href={`${chrome.getBasePath()}/app/ml#/jobs?mlManagement=(jobId:${id})`}
```

Comment:
I think you're okay without an `encodeURIComponent()` for the `jobId`.  As per the ML App create job flow, Job ID's are restricted to the following:


>Job ID can contain lowercase alphanumeric (a-z and 0-9), hyphens or underscores; must start and end with an alphanumeric character

Reply:
They might change that in the future though. I probably should add it as it is part of a URL.

Reply:
Added that and a test for the URI to have an encoding.